src/Tizen.NUI/src/public/WebView/WebBackForwardList.cs

```diff
@@ -62,7 +62,7 @@ public string Title
         /// <summary>
         /// Get original url.
```

Comment:
url -> URL

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520